.travis.yml

```
@@ -1,14 +1,20 @@
-language: java
+language: nix
 sudo: false
-jdk:
-  - oraclejdk8
 cache:
   directories:
-  - $HOME/.gradle/caches/
-  - $HOME/.gradle/wrapper/
+  - "$HOME/nix.store"
+  - "$HOME/.gradle/caches/"
+  - "$HOME/.gradle/wrapper/"
+before_install:
+- sudo mkdir -p /etc/nix
+- echo "substituters = https://cache.nixos.org/ file://$HOME/nix.store" | sudo tee -a /etc/nix/nix.conf > /dev/null
```

Wouldn't this do the same?
```suggestion
- sudo echo "substituters = https://cache.nixos.org/ file://$HOME/nix.store" >> /etc/nix/nix.conf
```
Why insert a `tee` only to discard the output stream with `> /dev/null`. Am I missing something…?